CRM/Event/ICalendar.php

```diff
@@ -44,7 +44,7 @@ public static function run() {
     $rss = CRM_Utils_Request::retrieveValue('rss', 'Positive', 0);
     $gCalendar = CRM_Utils_Request::retrieveValue('gCalendar', 'Positive', 0);
 
-    $info = CRM_Event_BAO_Event::getCompleteInfo($start, $type, $id, $end);
+    $info = CRM_Event_BAO_Event::getCompleteInfo($start, $type, $id, $end, FALSE);
```

Comment:
I'm not clear on this change. Is the PR also suggesting to make the feed itself start including non-public events?

Reply:
I don't think it does because this is specific for iCalendar generation of links not the feed I don't think. We have to add FALSE or else the info isn't returned if the Event is not active, which is the issue in the first place. Unless I missed something about feeds...

Reply:
@larssandergreen @demeritcowboy I see we have the `$rss`  so maybe I could use that for the `getCompleteInfo` like

```
CRM_Event_BAO_Event::getCompleteInfo($start, $type, $id, $end, !$rss);
```

Reply:
Nevermind I seperated it so it's more readable.. @demeritcowboy can you check that out and tell me what you think? I have never used the feed tbh

Reply:
Or maybe it's not really a privacy issue since you can do the same thing now with `civicrm/event/info?id=[guess]`. The checkbox for "public" here doesn't imply "not public" === "private".